config/routes.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+
+
+Rails.application.routes.draw do
+
+
+
+  resources :posts
+  #  only: [:new, :create, :index] 
+
+
+
+
+
+
+
+  devise_for :users, :controllers => {registrations: 'registrations' }
+
+
+
+
+
+
+
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+
+
+
+
+
+
+  root to: 'posts#index'
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+end
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
- [ ] Kindly remove all extra spaces in this file
- [ ] As the two previous TSE's noted ,the Post resource should only have an index, new, and create RESTful routes. i.e
`resources :posts, only: [:new, :create, :index]`  as described here
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))